src/components/common/DigitsEditor.vue

```diff
@@ -0,0 +1,70 @@
+<template>
```

Comment:
@object-required , I think `KeyDigitsSelector` name would fit better, your thoughts?

- [ ] rename if you agree